sys-process/audit/files/audit-rules.tmpfiles

```diff
@@ -1,5 +1,6 @@
-d   /etc/audit                          -   -   -   -   -
-d   /etc/audit/rules.d                  -   -   -   -   -
-L   /etc/audit/rules.d/00-clear.rules   -   -   -   -   /usr/share/audit/rules.d/00-clear.rules
-L   /etc/audit/rules.d/80-selinux.rules -   -   -   -   /usr/share/audit/rules.d/80-selinux.rules
-L   /etc/audit/rules.d/99-default.rules -   -   -   -   /usr/share/audit/rules.d/99-default.rules
+d   /etc/audit                          750   -   -   -   -
+L   /etc/audit/auditd.conf              640  -   -   -   /usr/share/auditd/auditd.conf
```

Comment:
```suggestion
C   /etc/audit/auditd.conf              640  -   -   -   /usr/share/auditd/auditd.conf
```
This will create the file if it doesn't exist. Unfortunately this means we won't update it but hopefully we soon find a way to do so.